config.yaml

```diff
@@ -51,6 +51,7 @@ orgs:
     - tirun
     - tklueber
     - trichie
+    - Trida
```

Comment:
You probably want to have "Trieda" here instead of "Trida", see https://github.com/Trieda